source/common/utility.cc

```diff
@@ -72,4 +72,10 @@ void Utility::parseCommand(TCLAP::CmdLine& cmd, const int argc, const char* cons
   }
 }
 
-} // namespace Nighthawk
\ No newline at end of file
+bool Utility::parseHostPort(const std::string& host_port, std::string* address, int* port) {
+  return RE2::FullMatch(host_port, R"((\d+\.\d+\.\d+\.\d+):(\d+))", address, port) ||
```

Comment:
Did we get these REs from somewhere or are they created by us? If we got them from somewhere, can we include a comment linking there? If they are crafted by us, can we add a comment above each one including an example string they are intending to match? That tends to help the posterity a lot when reading REs.